### PR TITLE
i18n(hi): add project-structure.mdx

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description (required)
- translate [project-structure](https://docs.astro.build/hi/basics/project-structure/) in Hindi language

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/hi/basics/project-structure.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+ब्राउज़र पर भेजी जाने वाली अंतिम वेबसाइट बनाने के लिए Astro आपकी `src/` फ़ाइलों को संसाधित, अनुकूलित और बंडल करता है। स्थिर `public/` निर्देशिका के विपरीत, आपकी `src/` फ़ाइलें Astro द्वारा आपके लिए बनाई और प्रबंधित की जाती हैं।
+
+कुछ फ़ाइलें (जैसे Astro अवयव) लिखित रूप में ब्राउज़र पर भी नहीं भेजी जाती हैं, बल्कि उन्हें स्थिर HTML में प्रस्तुत किया जाता है। अन्य फ़ाइलें (जैसे CSS) ब्राउज़र पर भेजी जाती हैं लेकिन प्रदर्शन के लिए उन्हें अन्य CSS फ़ाइलों के साथ अनुकूलित या बंडल किया जा सकता है।
+
+:::tip
+हालाँकि यह मार्गदर्शिका Astro समुदाय में उपयोग की जाने वाली कुछ लोकप्रिय परंपराओं का वर्णन करती है, Astro द्वारा आरक्षित एकमात्र निर्देशिकाएँ `src/pages/` और `src/content/` हैं। आप किसी भी अन्य निर्देशिका का नाम बदलने और उसे इस तरह से पुनर्व्यवस्थित करने के लिए स्वतंत्र हैं जो आपके लिए सर्वोत्तम हो।
+:::
+
+### `src/assets`
```
Comment (Mayank-Malviya) [inline src/content/docs/hi/basics/project-structure.mdx:69]: I didn't found this is [en/basics/project-structure](https://docs.astro.build/en/basics/project-structure/#src). Is there a reason why this is included ? Open question to other languages as well

File: src/content/docs/hi/basics/project-structure.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+### `astro.config.mjs`
+
+यह फ़ाइल प्रत्येक स्टार्टर टेम्पलेट में जेनरेट होती है और इसमें आपके Astro परियोजना के लिए कॉन्फ़िगरेशन विकल्प शामिल होते हैं। यहां आप उपयोग के लिए एकीकरण, निर्माण विकल्प, सर्वर विकल्प और बहुत कुछ निर्दिष्ट कर सकते हैं।
+
+कॉन्फ़िगरेशन सेट करने के विवरण के लिए [कॉन्फ़िगरिंग Astro मार्गदर्शिका](/hi/guides/configuring-astro/) देखें।
+
+### `tsconfig.json`
+
+यह फ़ाइल प्रत्येक स्टार्टर टेम्पलेट में उत्पन्न होती है और इसमें आपके Astro परियोजना के लिए TypeScript कॉन्फ़िगरेशन विकल्प शामिल होते हैं। कुछ सुविधाएँ (जैसे npm पैकेज आयात) `tsconfig.json` फ़ाइल के बिना संपादक में पूरी तरह से समर्थित नहीं हैं।
```
Comment (Mayank-Malviya) [inline src/content/docs/hi/basics/project-structure.mdx:129]:
```suggestion
यह फ़ाइल प्रत्येक स्टार्टर टेम्पलेट में उत्पन्न होती है और इसमें आपके Astro परियोजना के लिए TypeScript कॉन्फ़िगरेशन विकल्प शामिल होते हैं। कुछ सुविधाएँ (जैसे npm पैकेज आयात) `tsconfig.json` फ़ाइल के बिना एडिटर में पूरी तरह से समर्थित नहीं हैं।
```

**Review by Mayank-Malviya — CHANGES_REQUESTED** [review]
You missed some translations in the PR, I am sharing a suggestion for that.

File: src/content/docs/hi/basics/project-structure.mdx (hunk 1 above)
Comment (at-the-vr) [reply src/content/docs/hi/basics/project-structure.mdx:69]: your right, what I did was take a glance in [es/basics/project-structure](https://docs.astro.build/es/basics/project-structure/#srcassets), took an assumption that it's a "special information content". I will remove `assets` 👍

File: src/content/docs/hi/basics/project-structure.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+कुछ फ़ाइलें (जैसे Astro अवयव) लिखित रूप में ब्राउज़र पर भी नहीं भेजी जाती हैं, बल्कि उन्हें स्थिर HTML में प्रस्तुत किया जाता है। अन्य फ़ाइलें (जैसे CSS) ब्राउज़र पर भेजी जाती हैं लेकिन प्रदर्शन के लिए उन्हें अन्य CSS फ़ाइलों के साथ अनुकूलित या बंडल किया जा सकता है।
+
+:::tip
+हालाँकि यह मार्गदर्शिका Astro समुदाय में उपयोग की जाने वाली कुछ लोकप्रिय परंपराओं का वर्णन करती है, Astro द्वारा आरक्षित एकमात्र निर्देशिकाएँ `src/pages/` और `src/content/` हैं। आप किसी भी अन्य निर्देशिका का नाम बदलने और उसे इस तरह से पुनर्व्यवस्थित करने के लिए स्वतंत्र हैं जो आपके लिए सर्वोत्तम हो।
+:::
+
+### `src/assets`
+
+Astro द्वारा संसाधित की जाने वाली संपत्तियों (जैसे छवियों) को संग्रहीत करने के लिए [`src/assets`](/hi/guides/images/) फ़ोल्डर अनुशंसित फ़ोल्डर है। इसकी आवश्यकता नहीं है, और यह कोई विशेष आरक्षित फ़ोल्डर नहीं है.
+
```
Comment (at-the-vr) [inline src/content/docs/hi/basics/project-structure.mdx:72]:
```suggestion
```

File: src/content/docs/hi/basics/project-structure.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- [Markdown](/hi/guides/markdown-content/)
+
+ब्राउज़र पर भेजी जाने वाली अंतिम वेबसाइट बनाने के लिए Astro आपकी `src/` फ़ाइलों को संसाधित, अनुकूलित और बंडल करता है। स्थिर `public/` निर्देशिका के विपरीत, आपकी `src/` फ़ाइलें Astro द्वारा आपके लिए बनाई और प्रबंधित की जाती हैं।
+
+कुछ फ़ाइलें (जैसे Astro अवयव) लिखित रूप में ब्राउज़र पर भी नहीं भेजी जाती हैं, बल्कि उन्हें स्थिर HTML में प्रस्तुत किया जाता है। अन्य फ़ाइलें (जैसे CSS) ब्राउज़र पर भेजी जाती हैं लेकिन प्रदर्शन के लिए उन्हें अन्य CSS फ़ाइलों के साथ अनुकूलित या बंडल किया जा सकता है।
+
+:::tip
+हालाँकि यह मार्गदर्शिका Astro समुदाय में उपयोग की जाने वाली कुछ लोकप्रिय परंपराओं का वर्णन करती है, Astro द्वारा आरक्षित एकमात्र निर्देशिकाएँ `src/pages/` और `src/content/` हैं। आप किसी भी अन्य निर्देशिका का नाम बदलने और उसे इस तरह से पुनर्व्यवस्थित करने के लिए स्वतंत्र हैं जो आपके लिए सर्वोत्तम हो।
+:::
+### `src/components`
```
Comment (at-the-vr) [inline src/content/docs/hi/basics/project-structure.mdx:68]:
```suggestion

### `src/components`
```

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job 🙌

**Pull request merged**